specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -271,6 +402,171 @@
         }
       }
     },
+    "MultiLanguageConversationAnalysisInput": {
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Conversation"
+          }
+        }
+      }
+    },
+    "Conversation": {
+      "required": [
+        "id",
+        "language",
+        "modality"
+      ],
+      "properties": {
+        "id": {
+          "description": "Identifier for the conversation",
+          "type": "string"
+        },
+        "language": {
+          "description": "Default languages of the conversation item in BCP-47 format",
+          "type": "string"
+        },
+        "modality": {
+          "$ref": "#/definitions/Modality"
+        },
+        "domain": {
+          "type": "string"
+        },
+        "conversationItems": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationItem"
+          }
+        }
+      }
+    },
+    "Modality": {
+      "type": "string",
+      "description": "Enumeration of supported conversation genres (or modalities). Currently we will support chat and transcription",
+      "enum": [
+        "transcript",
+        "text"
+      ],
+      "x-ms-enum": {
+        "name": "Modality",
+        "modelAsString": true
+      }
+    },
+    "ConversationItem": {
+      "discriminator": "modality",
+      "required": [
+        "id",
+        "participantId",
+        "modality"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique identifier for the conversation item."
+        },
+        "participantId": {
+          "type": "string",
+          "description": "Identifier for the participant."
+        },
+        "language": {
+          "type": "string",
+          "description": "Optional property to override the language of the conversation item in BCP-47 format."
+        },
+        "modality": {
+          "description": "Property to override the modality of the conversation item.",
+          "$ref": "#/definitions/Modality"
+        }
+      }
+    },
+    "TextConversationItem": {
+      "type": "object",
+      "description": "Extra properties for supporting additional parameters for transcript conversation.",
+      "required": [
+        "text"
+      ],
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The text input to be analyzed"
+        }
+      },
+      "x-ms-discriminator-value": "text"
```

Comment:
This looks wrong. The `text` field should not be the discriminator. 
In fact, I think the discriminator should be in the base class, not here.